Update link to build page

benjaminaaron opened the pull request:
It's https://status.im/technical/build_status/ meanwhile, not https://status.im/build_status/ anymore.

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 5c8aa978 | [#1](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-10636/1/) | 2020-05-13 13:22:08 | ~8 min | `android-e2e` | [:package:`apk`](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200513-131325-5c8aa9-x86.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-prs.ams3.digitaloceanspaces.com%2FStatusIm-200513-131325-5c8aa9-x86.apk)|

flexsurfer commented:
@benjaminaaron thanks! there is a conflict, could you please rebase onto develop and resolve conflicts

flexsurfer commented on README.md:
```diff
@@ -33,7 +33,7 @@ John Dewey once said "Education is not preparation for life; education is life *
 
 - **Testers**  
-It's bug hunting season! Status is currently under active development and there is sure to be a bunch of learning, [build status from scratch](https://status.im/build_status/) or if an android user check out our [nightly builds](https://status.im/nightly). You can shake your phone to submit bug reports, or start browsing our [Github Issues](https://github.com/status-im/status-react/issues). Every ...
+It's bug hunting season! Status is currently in Alpha and there is sure to be a bunch of learning, [build status from scratch](https://status.im/technical/build_status/) or if an android user check out our [nightly builds](https://status.im/nightly). You can shake your phone to submit bug reports, or start browsing our [Github Issues](https://github.com/status-im/status-react/issues). Every bug y ...
```
@benjaminaaron thanks for the quick update, but you've reverted conflicted change, it should be `under active development ` instead `in Alpha` thanks

benjaminaaron replied:
Ups 🙈 I forced push that commit again, should be ok now?

benjaminaaron committed:
Update link to build page

It's https://status.im/technical/build_status/ meanwhile, not https://status.im/build_status/ anymore.

Signed-off-by: Andrey Shovkoplyas <motor4ik@gmail.com>
